libs/src/api/rewards.js

```diff
@@ -285,6 +287,7 @@ class Rewards extends Base {
           discoveryNodeAttestations: null,
           aaoAttestation: null,
           error: aaoAttestationError,
+          errorCode: errorCode,
```

Comment:
punning :)

Also, IMO this function should return aaoErrorCode rather than errorCode - otherwise it's unclear if this error refers to AAO or DN, both of which are involved in aggregating attestations